Makefile

```diff
@@ -254,8 +254,10 @@ ifneq ($(ADD_LDFLAGS), NONE)
 	LDFLAGS += $(ADD_LDFLAGS)
 endif
 
-ifneq ($(USE_CUDA_PATH), NONE)
-	NVCC=$(USE_CUDA_PATH)/bin/nvcc
+ifeq ($(NVCC), NONE)
```

Comment:
This change introduces the compile error when `nvcc` command is not defined in global scope. In that case, `$(NVCC)` is not `NONE`, but since `nvcc` is not defined in global scope, the compile would fail.

Reply:
A follow up PR logs more verbose warnings when this happens, and does select /usr/local/cuda/bin/nvcc as a fall back when nvcc isn't on the path.  The original implementation overwrote any value put into nvcc which would break compilation in many use cases when trying to cross compile.  The nvcc option was ignored in these cases and it still would just use whichever nvcc was your system default.